sentry-ruby/lib/sentry/client.rb

```diff
@@ -88,8 +88,10 @@ def send_event(event, hint = nil)
       event
     rescue => e
       loggable_event_type = (event_type || "event").capitalize
-      log_error("#{loggable_event_type} sending failed: #{e.message}")
-      log_error("Unreported #{loggable_event_type}: #{Event.get_log_message(event.to_hash)}")
+      log_error("#{loggable_event_type} sending failed", e, debug: configuration.debug)
+
+      event_info = Event.get_log_message(event.to_hash)
```

Comment:
@rhcarvalho this is not full event payload. you can see the old code does the exact same thing and I just moved it to a new line.
what it does is extracting the event's message attribute **or** the exception's name and no more. here's an example

```
Unreported Event: ZeroDivisionError: divided by 0
```

Reply:
Aha, sorry, I was mislead by the name 😢